WordPress/src/main/res/values/strings.xml

```diff
@@ -1001,6 +1002,7 @@
     <string name="notifications_push_summary">Settings for notifications that appear on your device.</string>
     <string name="search_sites">Search sites</string>
     <string name="notifications_no_search_results">No sites matched \'%s\'</string>
+    <string name="notification_sound_has_invalid_path">Invalid sound path. Please choose another.</string>
```

Comment:
Can we add something friendlier, like `"The sound you have chosen has an invalid file path. Please choose another one."`

Reply:
+1 for anything friendlier but, this is a Toast and the shorter the better since it goes away automatically. 100% open to suggestions that keep the work count low though :)

Reply:
We can also use `ToastUtils.showToast(context, text, Duration.LONG);` to make it live a little longer. Or maybe keep it a short duration and use something like `"The chosen sound has an invalid path. Please choose another."`, which is slightly friendlier and also just slightly longer

Reply:
On board with those suggestions! Done with bee96f8, 4fc7e3b.